service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
+
+echo "setting up test environment"
+
+# Set the working directory to the script's directory
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd -P)
+cd $script_dir
+echo "Generate busybox rootfs directory for runC tests"
+rootfs=runtime/runc/testbundle/rootfs
+mkdir -p $rootfs
+sudo mkdir -p /var/run/gcsrunc
+sudo docker export $(sudo docker create busybox) | tar -C $rootfs -xvf - > /dev/null
+
+echo "clone opencontailers/runc repo and build runc binary"
+
+cd $GOPATH/src/github.com
+mkdir opencontainers && cd opencontainers
+git clone https://github.com/opencontainers/runc runc
+cd runc && git checkout 992a5be178a62e026f4069f443c6164912adbf09
+make BUILDTAGS=``
+sudo cp ./runc /bin/runc
+/bin/runc -version
+
+echo "install Ginkgo and Gomega"
+cd $GOPATH/src/github.com
+go get github.com/onsi/ginkgo/ginkgo
+go get github.com/onsi/gomega
```

Comment:
You don't need to install gomega. Or cd into the $GOPATH/src/github.com directory. Also, this is handled by setup_test_env anyway.